README.rst

```diff
@@ -99,55 +102,106 @@ be found in the ``_build/html`` subdirectory.  PDFs can be built with the
 
 The openSUSE way
 ~~~~~~~~~~~~~~~~
-* sudo zypper in python-Sphinx
-* sudo zypper in python-rst2pdf
-* sudo zypper in python-sphinxcontrib-phpdomain # requires repository "devel:languages:python"
-* sudo zypper in pdfjam   # pull in latexpdf and all of texlive
-* sudo zypper in texlive-threeparttable
-* sudo zypper in texlive-wrapfig
-* sudo zypper in texlive-multirow
-* cd user_manual
-* make latexpdf
-* okular _build/latex/ownCloudUserManual.pdf
+
+First add the repository "devel:languages:python". How 
+this is done depends on your installation of openSUSE and the hardware 
+architecture. For detailed instructions, refer to `the SUSE documentation <https://software.opensuse.org/download.html?project=devel:languages:python&package=bpython>`_. 
+As an example, if you are using openSUSE 42.1, then you would run the following
+commands::
+
+  sudo zypper addrepo http://download.opensuse.org/repositories/devel:languages:python/openSUSE_Leap_42.1/devel:languages:python.repo \
+    && sudo zypper refresh
+
+After that, install the base dependencies, by running the following commands::
+
+  sudo zypper in python-Sphinx python-rst2pdf python-sphinxcontrib-phpdomain \
+    texlive-pdfjam texlive-threeparttable texlive-wrapfig texlive-multirow
+
+Alternatively, you can run ``./bin/unix/install-dependencies/opensuse.sh``.
 
 The Debian/Ubuntu way
 ~~~~~~~~~~~~~~~~~~~~~
-* sudo apt-get install python-pil
-* sudo apt-get install python-sphinx
-* sudo apt-get install python-sphinxcontrib.phpdomain
-* sudo apt-get install rst2pdf
-* sudo apt-get install texlive-fonts-recommended
-* sudo apt-get install texlive-latex-extra
-* sudo apt-get install texlive-latex-recommended
-* cd user_manual
-* make latexpdf
-* evince _build/latex/ownCloudUserManual.pdf
+
+To build the manual, first install the base dependencies, by 
+running the following commands::
+
+  sudo apt-get install python-pil python-sphinx python-sphinxcontrib.phpdomain \
+    rst2pdf texlive-fonts-recommended texlive-latex-extra texlive-latex-recommended
+
+Alternatively, you can run ``./bin/unix/install-dependencies/debian-ubuntu.sh``.
 
 The Arch Linux way
 ~~~~~~~~~~~~~~~~~~
-* sudo pacman -S community/python2-rst2pdf
-* sudo pacman -S community/python2-sphinx
-* aur/sphinxcontrib-phpdomain from AUR
-* sudo pacman -S extra/texlive-core texlive-latexextra
-* cd user_manual
-* make latexpdf
-* PDFVIEWER _build/latex/ownCloudUserManual.pdf
+
+To build the manual, first install the base dependencies, by 
+running the following commands::
+
+  sudo pacman-key --noconfirm --refresh-keys && \ 
```

Comment:
These lines shouldn't be joined with command chaining that makes them hard to copy and paste. I understand the logic, but when showing commands in documentation and expecting them to be run, the user should be responsible for noting whether they worked or not. Many folks have trouble with multiline editing in a terminal, so a format that runs one line at a time should be preferred here.